### ISSUE TITLE
Import Fail

### ISSUE THREAD

**Issue by EinsteinV** [assimp/assimp#2946]
Assimp::Importer::ReadFile fails on pFile with special characters like ö, é, or î etc. For example: D:/Földer/Tést/Fîle.stl

It does not fail when exporting to such a folder (this appears to be fixed, because in version 4.1.0 it could not do importing and exporting of such paths).

My guess it has something to do with UTF vs Latin1 vs UNIcode

**Comment by kimkulling** [comment]
This one is caused by the fact that our filesystem expects ASCII-names at the monebt. Maybe a duplicate one.

**Comment by kimkulling** [comment]
Sorry for closing this, I was much too fast.

**Comment by EinsteinV** [comment]
Ye I get that, but there are/exist paths or filenames that are something else then ASCII. The thing is, it was fixed for exporting, but not for importing... I awesme lib like this should be able to handle all. The fix shouldn't be to hard?

**Comment by agerrius** [comment]
I have also encountered this issue since upgrading to version 5.0, before when I was using the older version (I think 4, might even have been 3) it was working fine with such characters in the path.

**Comment by tellypresence** [comment]
@EinsteinV Please attach 3d model file affected by this problem

**Comment by agerrius** [comment]
I don't think a test file is needed, just rename any model so that it has a special character in the file name should be enough the reproduce it.

**Comment by EinsteinV** [comment]
@tellypresence Indeed, as agerrius mentioned, you can just take any file and rename it to contain any of the special/problematic characters that I mentioned. Not sure which assimp version I'm currently using and if it is still an issue, I can check later if needed?

**Comment by tellypresence** [comment]
@agerrius Thank you for following up -- will close for now, please reopen if it recurs

**Comment by GeniusProgger** [comment]
I just tested with Assimp Version: 5.2.5 (Revision: 3264642295). The issue is no longer present.
